src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  return (a) => {
+    return (b) => {
```

Comment:
называй переменные так, чтоб они характеризовали значение, которое хранят
```suggestion
  return (firstValue) => {
    return (secondValue) => {
```